styles/linter-plus.less

```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```

Comment:
Strange, removing that `display: block` still works fine for me. Not worth picking it apart though, it's small and it works! Thanks for clarifying.